ts_scripts/install_dependencies.py

```diff
@@ -41,26 +49,30 @@ def install_jmeter(self):
 class Linux(Common):
     def __init__(self):
         super().__init__()
+        os.system(f"{self.sudo_cmd}apt-get update")
 
     def install_java(self):
-        os.system(f"{self.sudo_cmd}apt-get update")
         os.system(f"{self.sudo_cmd}apt-get install -y openjdk-11-jdk")
 
-    def install_node_packages(self):
-        os.system(f"{self.sudo_cmd}apt-get update")
-        os.system(f"{self.sudo_cmd}npm install -g newman newman-reporter-html markdown-link-check")
-
     def install_nodejs(self):
-        os.system(f"{self.sudo_cmd}apt-get update")
+        python_path = Path(sys.executable).resolve()
         os.system(f"{self.sudo_cmd}curl -sL https://deb.nodesource.com/setup_14.x | {self.sudo_cmd}bash -")
         os.system(f"{self.sudo_cmd}apt-get install -y nodejs")
+        os.system(f"{self.sudo_cmd}ln -sf {python_path} /usr/bin/python")
+        os.system(f"{self.sudo_cmd}ln -sf /usr/bin/pip3 /usr/bin/pip")
```

Comment:
This might have side affects if the user is running in a virtual env. Please verify.

Reply:
@maaquib: The nodejs installation doesn't impact Conda or any virtual env's python. Validated on virtualenv as well.

```
(pytorchenv) ubuntu@ip-172-31-22-254:~/serve$ which python
/home/ubuntu/pyenv/pytorchenv/bin/python
(pytorchenv) ubuntu@ip-172-31-22-254:~/serve$ which pip
/home/ubuntu/pyenv/pytorchenv/bin/pip
(pytorchenv) ubuntu@ip-172-31-22-254:~/serve$ python --version
Python 3.7.5
(pytorchenv) ubuntu@ip-172-31-22-254:~/serve$ git checkout build_fixes
Branch 'build_fixes' set up to track remote branch 'build_fixes' from 'origin'.
Switched to a new branch 'build_fixes'
(pytorchenv) ubuntu@ip-172-31-22-254:~/serve$ python ts_scripts/install_dependencies.py 
<snip : install_dependencies_logs>
(pytorchenv) ubuntu@ip-172-31-22-254:~/serve$ 
(pytorchenv) ubuntu@ip-172-31-22-254:~/serve$ 
(pytorchenv) ubuntu@ip-172-31-22-254:~/serve$ 
(pytorchenv) ubuntu@ip-172-31-22-254:~/serve$ which pip
/home/ubuntu/pyenv/pytorchenv/bin/pip
(pytorchenv) ubuntu@ip-172-31-22-254:~/serve$ which python
/home/ubuntu/pyenv/pytorchenv/bin/python
(pytorchenv) ubuntu@ip-172-31-22-254:~/serve$ python --version
Python 3.7.5
```